api/spec/services/validator/stylesheet_spec.rb

```diff
@@ -0,0 +1,159 @@
+require "rails_helper"
+
+RSpec.configure do |c|
+  c.include Helpers
+end
+
+RSpec.describe Validator::Stylesheet do
+  let(:scope_selector) { Validator::Constants::CSS_SCOPE_SELECTOR }
+  let(:validator) { Validator::Stylesheet.new() }
```

Comment:
Style/MethodCallParentheses: Do not use parentheses for method calls with no arguments.